### DISCUSSION TITLE
Quantity Quality and protein quantification

### DISCUSSION THREAD

**Discussion by ht-lau** [vdemichev/DiaNN#326] [Q&A]
Hi Vadim,

- In the report.tsv, there is a Quantity.Quality column. Is that value used as a filter for quantification? I often see a spike at ~ 0.8 when I plot the values as a histogram. Will you recommend not using the peptide/precursor quant if the quality is below 0.8?
- I have learned that the PG.Quantity is the top1 peptide intensity. Does that apply to PG.Normalized, i.e. it is the normalized top1 peptide intensity?
- For MaxLFQ, are peptides filtered by any of the FDR columns before summing up? I also noticed that proteins with only 1 peptide will still produce a MaxLFQ value, which is different from both MaxQuant and FragPipe. Is there a way to change this through the command line options?

Thank you.

**Comment by vdemichev** [comment]
Hi,

- You can use Quantity.Quality for filtering, but this is experimental and we ourselves don't use it.
- Yes, PG.Normalised is also Top 1. But we recommend not to use these quantities, but use MaxLFQ instead.
- Run-specific 1% precursor FDR only. When using MBR, this automatically means also 1% global precursor FDR. Yes, 1 peptide is fine, I don't think it makes sense to requrie >= 2 peptides for most scenarios. If necesssary, can always filter out proteins with just one peptide using R/Python.

Best,
Vadim
